lib/evmone/baseline.cpp

```diff
@@ -12,16 +12,21 @@ namespace evmone
 {
 JumpdestMap build_jumpdest_map(const uint8_t* code, size_t code_size)
 {
-    JumpdestMap m(code_size);
+    // Bitmask for PUSH instructions identification.
+    // It removes the lower 5 bits of a PUSH instruction with information about the data length.
+    // The remaining top 3 bits of every PUSH instruction is 0b011, i.e. 0x60, OP_PUSH1.
```

Comment:
This sentence confused me, but probably it's just me.

Reply:
Better now?